### PR TITLE
Standardize NumPy imports

### PR DESCRIPTION
**Pull request by QuLogic**
The convention is `import numpy as np`, so use that instead of the various other forms.

### CODE REVIEW

File: gui/freehand.py [hunk 1]
```diff
@@ -772,8 +772,8 @@ def _interpolate_p0_p1(self):
                 t, x, y = np[0:3]
                 p, xt, yt = spline_4p(
                     float(t - t0) / dt,
-                    array(pt0p[3:]), array(pt0[3:]),
-                    array(pt1[3:]), array(pt1n[3:])
+                    np.array(pt0p[3:]), np.array(pt0[3:]),
+                    np.array(pt1[3:]), np.array(pt1n[3:])
```
Comment (achadwick) [inline gui/freehand.py]: I can't merge this PR yet due to the variable shadowing here.

**Comment by achadwick** [comment]
@QuLogic 
Please can you rebase against master then make sure the test case still runs? Thanks.

File: gui/freehand.py (hunk 1 above)
Comment (QuLogic) [inline gui/freehand.py]: Oops, should be fixed now.

**Comment by achadwick** [comment]
Thanks, looking good. :+1:

**Pull request merged**